instrumentation/kafka/kafka-clients/kafka-clients-0.11/testing/src/main/java/io/opentelemetry/instrumentation/kafka/internal/KafkaClientBaseTest.java

```diff
@@ -219,24 +215,21 @@ protected static List<AttributeAssertion> processAttributes(
                 equalTo(SemanticAttributes.MESSAGING_DESTINATION_NAME, SHARED_TOPIC),
                 equalTo(SemanticAttributes.MESSAGING_OPERATION, "process"),
                 satisfies(
-                    SemanticAttributes.MESSAGING_KAFKA_CLIENT_ID,
+                    SemanticAttributes.MESSAGING_CLIENT_ID,
                     stringAssert -> stringAssert.startsWith("consumer")),
-                satisfies(
-                    SemanticAttributes.MESSAGING_KAFKA_SOURCE_PARTITION,
-                    AbstractLongAssert::isNotNegative),
+                // TODO (trask) does this have a replacement?
+                // satisfies(
+                //     SemanticAttributes.MESSAGING_KAFKA_SOURCE_PARTITION,
+                //     AbstractLongAssert::isNotNegative),
```

Comment:
@pyohannes @lmolkova @joaopgrassi I couldn't tell if it was intentional that there is no replacement for `messaging.kafka.source.partition`?

Reply:
was removed in https://github.com/open-telemetry/semantic-conventions/pull/100

Reply:
what I wasn't sure about is that https://github.com/open-telemetry/semantic-conventions/pull/156 introduces replacements for the other attributes removed in https://github.com/open-telemetry/semantic-conventions/pull/100

Reply:
opened https://github.com/open-telemetry/semantic-conventions/issues/300

Reply:
For the time being I think we should not remove that attribute we can just keep using `stringKey("messaging.kafka.source.partition")` until the situation clarifies.

Reply:
If I remember correctly, when we agreed to remove the `source` namespace, we bumped into that one and we couldn't come up with use cases where it was used in that way, but I agree we probably should clarify. I will add this to the agenda so it can be discussed in the next SIG meeting (tomorrow).